pymc3_hmm/step_methods.py

```diff
@@ -42,6 +59,10 @@
 from pymc3_hmm.distributions import DiscreteMarkovChain, HorseShoe, SwitchingProcess
 from pymc3_hmm.utils import compute_trans_freqs
 
+from functools import singledispatch
+from polyagamma import random_polyagamma
```

Comment:
This implies that `polyagamma` is now a package requirement.  Since that's not necessary, we can move this import closer to a location where it will be used, or we can catch the `ImportError` and ignore it.